pkg/leeway/build.go

```diff
@@ -1283,8 +1283,8 @@ func (p *Package) buildGo(buildctx *buildContext, wd, result string) (res *packa
 	if len(buildCmd) > 0 && cfg.Packaging != GoLibrary {
 		commands[PackageBuildPhaseBuild] = append(commands[PackageBuildPhaseBuild], buildCmd)
 	}
-	commands[PackageBuildPhaseBuild] = append(commands[PackageBuildPhaseBuild], []string{"rm", "-rf", "_deps"})
 
+	commands[PackageBuildPhasePackage] = append(commands[PackageBuildPhasePackage], []string{"rm", "-rf", "_deps"})
```

Comment:
Is this change relevant, or a cleanup? If yes: How is it relevant? :thinking:

Reply:
Saw the commit comment now. :+1:

Reply:
This is the change that actually fixes the go cover collection. It does that by removing the deps prior to packaging instead of directly after the build. This way the dependencies are still present when the go coverage collection happens.